common/array.h

```diff
@@ -141,6 +145,12 @@ class Array {
 		insert_aux(_storage + idx, array.begin(), array.end());
 	}
 
+	/**
+	 * Inserts element before pos.
+	 */
```

Comment:
It’s meant to be before, since that matches std::vector and Common::List and presumably insert_at. Is it not doing a before-insertion?